swift/ql/lib/codeql/swift/frameworks/Cryptography/CommonCrypto.qll

```diff
@@ -0,0 +1,126 @@
+/** Provides models and concepts from the CommonCrypto C library */
+
+private import swift
+private import codeql.swift.dataflow.DataFlow
+private import codeql.swift.dataflow.ExternalFlow
+private import codeql.swift.security.Cryptography
+
+/**
+ * A model for CommonCrypto functions that permit taint flow.
+ */
+private class CommonCryptoSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        // "namespace; type; subtypes; name; signature; ext; input; output; kind"
+        ";;false;CCCryptorCreate(_:_:_:_:_:_:_:);;;Argument[1];Argument[6];value",
+        ";;false;CCCryptorCreateFromData(_:_:_:_:_:_:_:_:_:_:);;;Argument[1];Argument[8];value",
+        ";;false;CCCryptorCreateWithMode(_:_:_:_:_:_:_:_:_:_:_:_:);;;Argument[1..2];Argument[10];value",
+      ]
+  }
+}
+
+/** A data flow node for the output of a CommonCrypto encryption/decryption operation */
+abstract private class CommonCryptoOutputNode extends CryptographicOperation::Range {
```

Comment:
I think this should just explicitly model `CCCrypt` and `CCCryptorUpdate`?  It looks like you have everything you need to figure out which algorithm is used (at which point you _should_ begin to get results on the tests), and I guess similar logic for the block mode.